### DISCUSSION TITLE
It's the best for the game

### DISCUSSION THREAD

**Discussion by Riuk1z** [nzp-team/nzportable#1032] [Other Content Releases]
It would also be good to have the game on Steam and add a workshop so as not to be accessing the game files and replacing the files because it is a very slow process since the workshop only takes one click and the mod is already installed and also add the Spanish as language

**Comment by BrownNPC** [comment]
I don't know if they can automate steam updates, cuz this game updates like twice a day

**Reply by BrownNPC** [reply]
Also I dont understand how it's difficult to install mods. All you gotta do is rename the zip file to PK3 and drag and drop it into the game
